paris-material/src/main/java/com/airbnb/paris/material/utils/ViewUtils.kt

```diff
@@ -0,0 +1,18 @@
+package com.airbnb.paris.material.utils
+
+import android.graphics.PorterDuff
+
+object ViewUtils {
+
+    fun parseTintMode(value: Int, defaultMode: PorterDuff.Mode): PorterDuff.Mode {
```

Comment:
Reiterating a comment above: this is already implemented by `ViewProxy` and is not needed here.